src/System.Runtime.Serialization.Primitives/tests/project.json

```diff
@@ -0,0 +1,41 @@
+{
+  "dependencies": {
+    "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0030",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24113-00",
+    "System.Collections.NonGeneric": "4.0.1-rc3-24113-00",
+    "System.Console": "4.0.0-rc3-24113-00",
+    "System.Globalization": "4.0.11-rc3-24113-00",
+    "System.Linq.Expressions": "4.1.0-rc3-24113-00",
+    "System.ObjectModel": "4.0.12-rc3-24113-00",
+    "System.Reflection": "4.1.0-rc3-24113-00",
+    "System.Reflection.Emit": "4.0.1-rc3-24113-00",
+    "System.Runtime.Extensions": "4.1.0-rc3-24113-00",
+    "System.Runtime.InteropServices": "4.1.0-rc3-24113-00",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc3-24113-00",
+    "System.Text.RegularExpressions": "4.1.0-rc3-24113-00",
+    "System.Threading": "4.0.11-rc3-24113-00",
+    "System.Threading.Tasks": "4.0.11-rc3-24113-00",
```

Comment:
Hmm, then I get

  C:\git\corefx.dan\Tools\packageresolve.targets(126,5): error : Dependency validation error: for System.Threading 4.0.
11-rc3-24117-00 in C:\git\corefx.dan\src\System.Runtime.Serialization.Primitives\tests/project.json package prerelease
is 'rc3-24117-00', but expected 'rc3-24113-00' for packages matching '^(?i)((System\..*)|(NETStandard\.Library)|(Micros
oft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.TargetingPack\.Private\.(CoreCLR|NETNative))|(Microsoft\.Win32\..*)|(Mi
crosoft\.VisualBasic))(?<!TestData)(?<!System\.Data\.SqlClient)$' [C:\git\corefx.dan\src\System.Runtime.Serialization.P
rimitives\tests\System.Runtime.Serialization.Primitives.Tests.csproj]